tests/remote-containers.sh

```diff
@@ -92,12 +93,23 @@ for image in "${!IMAGES[@]}"; do
         # TODO: Do we have to test all $(VERSION)s?
         # TODO: The PS4 hack doesn't work if we run the testsuite as UID=0.
         PS4="+ [$image] " make TARGET="$OS" test
-
+        if test $? -eq 0 ; then
+          printf -v test_short_summary "${test_short_summary}[PASSED] $image\n"
+          info "Tests for $image succeeded."
+        else
+          printf -v test_short_summary "${test_short_summary}[FAILED] $image\n"
+          info "Tests for $image failed."
+          TESTSUITE_RESULT=1
+        fi
```

Comment:
I do not think this branch will ever get used since you still have `set -e` set in the subshell (at line 57).